benchmarks/bench_higgs_boson.py

```diff
@@ -85,7 +85,7 @@ def load_data():
                                          max_leaf_nodes=n_leaf_nodes,
                                          n_iter_no_change=None,
                                          random_state=0,
-                                         verbose=1)
+                                         verbose=1, parallel_splitting=False)
```

Comment:
Maybe you can expose this a commandline flag to make it easy to profile with `mprof run benchmarks/bench_higgs_boson.py --disable-parallel-splitting` (using `memory_profiler`).